game/loops.py

```diff
@@ -1005,6 +1040,16 @@ def create_profile(clock, track, player_car, computer_car, game_info, player_pro
                     clock, track, player_car, computer_car, game_info, player_profile
                 )
 
+            elif (
+                models.Profanity.select()
+                .where(models.Profanity.word == name_entry_box.text.lower())
+                .exists()
+            ):
+                blit_text_center(WIN, MAIN_FONT, "Username cannot contain profanity!")
```

Comment:
Updated "cannot be profanity" to "cannot contain profanity" in a subsequent push.